.pre-commit-config.yaml

```diff
@@ -20,7 +21,17 @@ repos:
       - id: check-json
       - id: end-of-file-fixer
       - id: requirements-txt-fixer
+  - repo: https://github.com/pycqa/flake8
```

Comment:
Mainly for personal development reasons - I cannot run locally the provided docker image, so I updated the current local configuration.
This could be excluded from vcs, to keep the functionality local.